pkg/commonchecks/corechecks.go

```diff
@@ -78,6 +79,7 @@ func RegisterChecks(store workloadmeta.Component) {
 	corecheckLoader.RegisterCheck(network.CheckName, network.Factory())
 	corecheckLoader.RegisterCheck(nvidia.CheckName, nvidia.Factory())
 	corecheckLoader.RegisterCheck(oracle.CheckName, oracle.Factory())
+	corecheckLoader.RegisterCheck(oracledbm.CheckName, oracle.Factory())
```

Comment:
Would it be possible to place old check name in the new package? I don't think we need a separate package just for a single constant.

Reply:
Right, thanks. I changed it.